.github/workflows/release-upload.yml

```diff
@@ -198,13 +208,14 @@ jobs:
             tar -czf ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-$arch.tar.gz cardano-cli-$arch
           done
           # TODO generalize
-          # zip ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-win64.zip cardano-cli-win64
+          zip -r9 ${{ needs.wait_for_hydra.outputs.TARGET_TAG }}-win64.zip cardano-cli-win64
```

Comment:
What does `-r9` do?

Reply:
It is a combination of `-r` and `-9`:
- `-r` is like in `cp`, it means "recursively" and it is necessary because we are compressing a directory.
- `-9` means maximum compression level, the options are from `-0` (no compression), to `-9` (slowest compression)